smarts/core/vehicle_index.py

```diff
@@ -45,8 +46,12 @@ class _ControlEntity(NamedTuple):
     # Applies to shadowing and controlling actor
     # TODO: Consider moving this to an _ActorType field
     is_boid: bool
+    position: np.ndarray
 
 
+# TODO: Consider wrapping the controlled_by recarry into a sep state object
```

Comment:
Nothing particularly important; just considering another way to do VehicleIndex state diffing.